lib/libmarshal.map

```diff
@@ -256,6 +252,14 @@
         Tss2_MU_TPMT_TK_AUTH_Unmarshal;
         Tss2_MU_TPMT_TK_HASHCHECK_Marshal;
         Tss2_MU_TPMT_TK_HASHCHECK_Unmarshal;
+        Tss2_MU_TPMS_EMPTY_Marshal;
+        Tss2_MU_TPMS_EMPTY_Unmarshal;
+        Tss2_MU_TPM2_HANDLE_Marshal;
+        Tss2_MU_TPM2_HANDLE_Unmarshal;
+        Tss2_MU_TPM2_SE_Marshal;
+        Tss2_MU_TPM2_SE_Unmarshal;
+        Tss2_MU_TPMI_ALG_HASH_Marshal;
+        Tss2_MU_TPMI_ALG_HASH_Unmarshal;
```

Comment:
We also need to add BYTE MU definitions here

Reply:
fixed